cfg/std.cfg

```diff
@@ -8782,7 +8782,6 @@ initializer list (7) string& replace (const_iterator i1, const_iterator i2, init
       <member name="second" templateParameter="1"/>
     </rangeItemRecordType>
     <access>
-      <function name="at" yields="at_index"/>
```

Comment:
Was this intentional?

Reply:
Yes, `std::multimap` has not `.at()`
https://en.cppreference.com/w/cpp/container/multimap